### DISCUSSION TITLE
Is it possible to exit early without cancelling --watch

### DISCUSSION THREAD

**Discussion by is-jonreeves** [oven-sh/bun#10983] [Q&A]
I noticed that when I intentionally exit a script using `process.exit()` it cancels out of bun's watch mode. If I provide a string (eg...  `process.exit('')`) to imply an error then it keeps watching (same as using `throw new Error('')`).

This isn't affecting me in real-world code. While I'm drafting/testing ideas in a file I was just attempting to use `process.exit()` as a marker to avoid commenting out work below the fold.

I can see that this might be a purposeful behavior so was wondering if there is a more appropriate way to exit a script (perhaps I'm just missing a special exit code)?

**Comment by akodkod** [comment]
+1 for this issue
